docs/start/quickstart.md

```diff
@@ -1 +1,192 @@
 # Quickstart
+
+The programming model consists of wrapping the code containing the model training logic inside a function. 
```

Comment:
wrapping the code containing the model training logic inside a function ->
writing a Python function that includes the training logic, that we call the training function.